modules/standard/IO.chpl

```diff
@@ -430,7 +430,9 @@ module IO {
       (ie, they can open up channels that are not shared).
 */
 
-public use SysBasic;
+import SysBasic.{syserr,EFORMAT,fd_t,ENOERR,EEOF,qio_err_t};
+import OS.POSIX.{ENOENT, ENOSYS, EINVAL, EILSEQ, EIO, ERANGE};
+private import OS.POSIX.{EBADF};
```

Comment:
It seems weird that this one is `private import` when the above are just `import` (and import is private by default).

Reply:
Good point.  The `private` made more sense when the previous ones were `public`, but once I removed the need for them to be it seems silly.  This'll be removed in my next commit.